examples/Patterns/Saga/Program.cs

```diff
@@ -11,6 +11,8 @@ namespace Saga
 {
     internal class Program
     {
+        public static readonly RootContext Context = new RootContext();
```

Comment:
I have no idea if this is how context's should be managed. I will look through the docs to see if I can find out, however, given that this particular sample looks like a very high profile sample (because of the extra effort that appears to have gone into the accompanying readme documentation, I would really appreciate any extra eyes on this change to make sure the resulting fix is in line with coding best practices for protoactor.

My main concern with this, is that it's a global static, and I would expect that the context would be injected somehow during props constructions or whatever mechanisms protoactor uses.

Reply:
A new RootContext is really only needed if there's context that needs to be passed through middleware - correct me if I'm wrong @rogeralsing? :) Meaning you could skip declaring your own instance and just use RootContext.Empty instead.

Reply:
Done. `RootContext` is now `RootContext.Empty`